templates/python/.github/workflows/python-poetry-release.yaml

```diff
@@ -0,0 +1,27 @@
+name: Default reusable workflow
+
+on:
+  push:
+    branches: [main]
+
+jobs:
+  call-workflow-passing-data:
```

Comment:
please change the name
and add `name: ...` attribute

Reply:
Changed it. It's more specific than the default, but lmk if you think it doesn't fit. I just wrote the 3 steps that are taken.

Reply:
_release_ would be a more fitting name imo

Reply:
changed